code/modules/cargo/exports/xenobio.dm

```diff
@@ -0,0 +1,43 @@
+/datum/export/slime
```

Comment:
Big big personal gripe, slime cores themselves are, similar to plastic above, effectively infinite. Due to how crossbreeding works, and the fact that each slime splits into 3-5 different slimes each, you create an MINIMUM sustainable average of 450 credits each.  Now, the thing is, this isn't being given to science itself, it's all being given to the cargo budget, but since you've used a floor of 150 credits for grays, and a maximum of 1000 for high rarity slimes, this can easily, EASILY be abused for massive profits. It's for this reason that when I did foox exports, I kept most individual foods at a lower threshold for things that can be far too easily produced.

Reply:
Now, with this said, if you did more to keep exports as the unique exports of xenobiology, that will feel more rewarding and encourage sharing xenobiology rewards with the rest of the station (Potions, weapons, utility things, etc.)